test/cpp/jit/CMakeLists.txt

```diff
@@ -44,6 +44,10 @@ if(USE_CUDA)
   list(APPEND JIT_TEST_SRCS ${JIT_TEST_ROOT}/test_gpu.cpp)
 endif()
 
+if(USE_ROCM)
+  list(APPEND JIT_TEST_SRCS ${JIT_TEST_ROOT}/test_gpu.cpp)
```

Comment:
Do we actually execute any cpp tests for our CI?  Would this test file be covered?

Reply:
We don't run it. I would like to include it in the CI in this PR. But can't find `test_jit` binary.